teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>side note for NGLview, Jaime shared recently with me, to center the figures on the binding site per default one can use this: <code>v.center("ligand")</code> (see <a href="http://nglviewer.org/ngl/api/manual/selection-language.html#expressions" target="_blank">manual</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>